test/central_egress_ingress_no_tgw_association_propagation/central_egress_ingress_no_tgw_association_propagation_test.go

```diff
@@ -0,0 +1,95 @@
+package test
+
+import (
+	"testing"
+
+    "github.com/stretchr/testify/assert"
+
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+    "github.com/gruntwork-io/terratest/modules/random"
+)
+
+func TestCentralEgressIngressNoAssociationAndPropagation(t *testing.T) {
+    helperDir := "./test_helpers"
```

Comment:
for consistency across the codebase can you please rename this dir to `hcl_fixtures`

Reply:
Done. In all my tests I renamed folders and relevant variables.